### ISSUE TITLE
Capture changes needed in protocol to support the apps 

### ISSUE THREAD

**Issue by VJag** [atsign-foundation/at_client_sdk#470]
**Is your feature request related to a problem? Please describe.**
There have been requests coming from the app developers for the enhancements needed in the protocol. The objective of this task is to capture and analyze the requests and brainstorm them further.

**Describe the solution you'd like**
NA

**Describe alternatives you've considered**
NA

**Additional context**
[Doc](https://docs.google.com/document/d/1phswZbEYGjayzqquhsglidXNTQTPAeHIWsoVfIAmLW4/edit?usp=sharing)

**Comment by gkc** [comment]
There is much work happening on this front already (improvements to error handling, streaming sync) but keeping this ticket open as a place to capture further ideas for enhancement.

**Comment by ksanty** [comment]
@gkc is this an arch discussion item?

**Comment by gkc** [comment]
Yes. But particularly for IoT apps which are unattended by humans

**Comment by gkc** [comment]
Closing as stale / not planned after triage.

This was a 2022 catch-all bucket for "protocol changes the apps might need", with no scoped change — its only payload is an external Google Doc, so there's nothing concrete to action or verify against the code. The themes it was gathering have since moved to concrete homes: streaming sync (fsync) is tracked at #504, and the encryption / protocol-agility work is the post-quantum program (`docs/projects/pq/`, #1889 / #1891 / #1893).

If the linked doc still holds specific un-migrated requests, the right move is to raise them as individual issues rather than keep an open idea-bucket. Closing.
